src/components/editor/State.js

```diff
@@ -436,6 +435,8 @@ class ConditionOnset extends Component<Props> {
         <br />
         {this.renderAssignToAttribute()}
         <div className='section'>
+          Value Set: <RIEInput className='value-set-text editable-text' value={state.value_set || ''} propName={'value_set'} change={this.props.onChange('value_set')} />
```

Comment:
Careful, looks like your value set stuff got added here as well. If you're working on multiple tasks at the same time, it's good to use `git add -p <filename>` -- the `-p` flag meaning by parts so it will ask you to confirm each changed section of the file. That way you don't add anything unexpected to the commit.

Reply:
ah I forgot I had added that.  I'll take it out for now its just one line, but will use `-p` in the future